src/evaluate-script.ts

```diff
@@ -81,12 +91,15 @@ export function evaluateScript(source: string) {
 
   return async function(e: GraphQLEventFields): Promise<any | undefined> {
     let retPromise: ResolverResponse | undefined = undefined;
-    const event = {
```

Comment:
Is there a reason why this is not const? I don’t see where it is reassigned.

Reply:
Oh my bad, fixed. thanks!